addons/sale_project/static/tests/project_task_list_sale_line.test.js

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, test } from "@odoo/hoot";
+import { check, queryAll, queryOne } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { mountView } from "@web/../tests/web_test_helpers";
+import { mailModels } from "@mail/../tests/mail_test_helpers";
+import { defineProjectModels, projectModels } from "@project/../tests/project_models";
+import { ProjectTask } from "./project_task_model"
+
+describe.current.tags("desktop");
+defineProjectModels();
+
+test("cannot edit sale_line_id when partners are different", async () => {
+    mailModels.ResPartner._records = [
+        { id: 101, name: "Deco Addict" },
+        { id: 102, name: "Azure Interior" },
+        ...mailModels.ResPartner._records,
+    ];
+
+    ProjectTask._records = [
```

Comment:
we could do this no? 
```suggestion
    projectModels.ProjectTask._records = [
```

Reply:
> we could do this no?

It is possible. however, we are currently accessing it this way in project_enterprise. What is your opinion now?
[ref](https://github.com/odoo/enterprise/commit/f47a8c4f23b81d1a554885767817144dc8377033#diff-381338b2debce4f97fd60ab04b2de784fc7d914595e0d28cbd2e7a76ef055f43R8)